src/blockdev.rs

```diff
@@ -437,15 +437,19 @@ fn read_sysfs_dev_block_value(maj: u64, min: u64, field: &str) -> Result<String>
     Ok(read_to_string(&path)?.trim_end().into())
 }
 
-pub fn lsblk(dev: &Path) -> Result<Vec<HashMap<String, String>>> {
-    // Older lsblk, e.g. in CentOS 7.6, doesn't support PATH, but -p option
+pub fn lsblk(dev: &Path, with_deps: bool) -> Result<Vec<HashMap<String, String>>> {
+    // Older lsblk, e.g. in CentOS 7.6, doesn't support PATH, but --paths option
     let mut cmd = Command::new("lsblk");
     cmd.arg("--pairs")
         .arg("--paths")
         .arg("--output")
         .arg("NAME,LABEL,FSTYPE,TYPE,MOUNTPOINT")
         .arg(dev);
 
+    if !with_deps {
+        cmd.arg("--nodeps");
```

Comment:
Can't the caller just filter on `TYPE` == `disk`?

Reply:
Ah, I see you're running this on a partition.  The caller could filter on `NAME`, I guess, but that may not match the argument if a symlink is involved.  I'm okay leaving this as is.